packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml

```diff
@@ -79,15 +79,6 @@ patrol:
     bundle_id: {{domain_name}}.{{organization_name}}.{{project_name}}
     app_name: {{project_name}} {{/enable_patrol}}
 
-
-  {{#enable_dev_menu}}
-dependency_overrides:
```

Comment:
It looks like the fork is not needed anymore. Should we delete it from our account ?

Reply:
Yes, we can proceed with removing it.